### PR TITLE
Validium 

### PR DESCRIPTION
**Pull request by begmaroman**

### CODE REVIEW

File: sequencesender/sequencesender.go [hunk 1]
```diff
@@ -456,8 +458,15 @@ func (s *SequenceSender) tryToSendSequence(ctx context.Context) {
...
+	// Post sequences to DA backend
+	dataAvailabilityMessage, err := s.da.PostSequence(ctx, sequences)
+	if err != nil {
+		log.Error("error posting sequences to the data availability protocol: ", err)
+		return
+	}
+
 	// Build sequence data
-	to, data, err := s.etherman.BuildSequenceBatchesTxData(s.cfg.SenderAddress, sequences, uint64(lastSequence.LastL2BLockTimestamp), firstSequence.BatchNumber-1, s.cfg.L2Coinbase)
+	to, data, err := s.etherman.BuildSequenceBatchesTxData(s.cfg.SenderAddress, sequences, uint64(lastSequence.LastL2BLockTimestamp), firstSequence.BatchNumber-1, s.cfg.L2Coinbase, dataAvailabilityMessage)
```
Comment (zjg555543) [inline sequencesender/sequencesender.go]: Great PR for validium. Does it need to support both rollup and validium mode?

Comment (begmaroman) [reply sequencesender/sequencesender.go]: Yes, I'll have to add more logic to be able to run it in two different modes.

**Review by vcastellm — COMMENTED** [review]
Could we use https://github.com/0xPolygon/cdk-contracts-tooling for the contracts interaction?

File: sequencesender/sequencesender.go (hunk 1 above)
Comment (begmaroman) [reply sequencesender/sequencesender.go]: Done

**Comment by begmaroman** [comment]
Merging this PR into `validium` branch to continue testing. There will be another PR from `validium` into `develop` right after everything is done.

**Pull request merged**